pillarbox-demo-cast/src/main/java/ch/srgssr/pillarbox/demo/cast/MainActivity.kt

```diff
@@ -11,15 +12,14 @@ import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.tooling.preview.Preview
-import androidx.fragment.app.FragmentActivity
 import ch.srgssr.pillarbox.cast.getCastContext
 import ch.srgssr.pillarbox.cast.widget.CastButton
 import ch.srgssr.pillarbox.demo.cast.ui.theme.PillarboxTheme
 
 /**
  * Activity showing how to use Cast with Pillarbox.
  */
-class MainActivity : FragmentActivity() {
+class MainActivity : ComponentActivity() {
```

Comment:
Cast UI have to be a FragmentActivity otherwise when clicking cast button it crashes.

Reply:
Ah, good catch.
I've switched back to using `FragmentActivity`